packages/engine-server/src/config.ts

```diff
@@ -48,6 +48,9 @@ export class DConfig {
       mermaid: true,
       useKatex: true,
       autoFoldFrontmatter: true,
+      dev: {
+        enablePreviewV2: true,
```

Comment:
should we just remove this setting altogether?

Reply:
i was going to give preview v1 at least another two weeks before removing it altogether as an option